src/Microsoft.ML.Data/Data/SchemaDefinition.cs

```diff
@@ -332,7 +332,16 @@ public enum Direction
         /// <param name="userType">The type to base the schema on.</param>
         /// <param name="direction">Accept fields and properties based on their direction.</param>
         /// <returns>The generated schema definition.</returns>
-        public static SchemaDefinition Create(Type userType, Direction direction = Direction.Both)
+        public static SchemaDefinition Create(Type userType, Direction direction = Direction.Both) => Create(userType, direction, null);
+
+        /// <summary>
+        /// Create a schema definition by enumerating all public fields of the given type.
+        /// </summary>
+        /// <param name="userType">The type to base the schema on.</param>
+        /// <param name="direction">Accept fields and properties based on their direction.</param>
+        /// <param name="data">onnx data information</param>
+        /// <returns>The generated schema definition.</returns>
```

Comment:
IDataView is not related to Onnx. It is one of the central interfaces of  ML.NET and encapsulates a view of the data that is used as the input and output of transformers. See (here)[[https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.idataview?view=ml-dotnet](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.idataview?view=ml-dotnet)]

Reply:
Thanks for the link!